src/app/workflow/workflow.component.html

```diff
@@ -222,18 +222,41 @@ <h3>
               >
               </app-versions-workflow>
             </div>
-            <div *ngIf="isStub()" class="p-3">
-              <mat-card class="alert alert-warning" role="alert">
-                <mat-icon>warning</mat-icon> To see versions, please refresh the workflow.
-              </mat-card>
-            </div>
+            <ng-template #noVersionsVersionTab>
+              <div class="p-3">
+                <mat-card class="alert alert-info" role="alert">
+                  <mat-icon>info</mat-icon>
+                  <span *ngIf="entryType === EntryType.Service; else noVersionsVersionTabBioWorkflow">
+                    This service does not have any versions. Create a release/tag on GitHub with a valid dockstore config file to add a new
+                    version.
+                  </span>
+                  <ng-template #noVersionsVersionTabBioWorkflow>
+                    <span *ngIf="workflow?.mode === WorkflowModel.ModeEnum.HOSTED">
+                      To see versions, please add a new version.
+                    </span>
+                    <span *ngIf="workflow?.mode !== WorkflowModel.ModeEnum.HOSTED">
+                      To see versions, please refresh the workflow.
+                    </span>
+                  </ng-template>
+                </mat-card>
+              </div>
+            </ng-template>
           </mat-tab>
           <mat-tab id="fileTab" label="Files">
             <ng-template matTabContent>
-              <div *ngIf="entryType === EntryType.Service; else BioWorkflow">
-                <app-entry-file-tab></app-entry-file-tab>
+              <div *ngIf="entryType === EntryType.Service; else FilesTabBioWorkflow" class="p-3">
+                <div *ngIf="(extendedWorkflow$ | async)?.workflowVersions.length > 0; else noVersionsFilesTab">
+                  <app-entry-file-tab></app-entry-file-tab>
+                </div>
+
+                <ng-template #noVersionsFilesTab>
+                  <mat-card class="alert alert-info" role="alert">
+                    <mat-icon>info</mat-icon> This service does not have any versions. Create a release/tag on GitHub with a valid dockstore
```

Comment:
Instead of `dockstore config file`, may you should say `.dockstore.yml`. Or leave `dockstore config file`, and also add `.dockstore.yml` -- I don't know that new users know what a dockstore config file is.